en/add_wagtail_to_your_website/install_wagtail/README.md

```diff
@@ -0,0 +1,179 @@
+# Wagtail Installation and Configuration
+
+> Part of this chapter is based on the WagTail tutorial by 
+> TorchBox (https://docs.wagtail.org/en/stable/getting_started/integrating_into_django.html).
+
+In this section, we will cover how to install and configure Wagtail in an already existing project. The experience will
+slightly be different from just launching a new Wagtail project as we already have a working project that we need to 
+improve by adding the powerful and wonderful capabilities of Wagtail.
+
+## Wagtail Installation
+To install Wagtail, type the following command in the command prompt:
+
+```
+(myvenv) ~/djangogirls$ pip install wagtail
```

Comment:
Can confirm this will break at some point, as Wagtail 6.0 due for release in February 2024 will drop Django 3.2 compatibility. So for now I’d recommend:

```suggestion
(myvenv) ~/djangogirls$ pip install wagtail>=5.2,<5.3
```